portfolio/src/main/webapp/script.js

```diff
@@ -26,13 +26,86 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
 async function getQuote(){
     const response = await fetch('/quoteServ');
     const quote = await response.text();
     document.getElementById('quote-container').innerText = quote;
 }
+
 async function getMessage(){
     const response = await fetch('/messageServ');
     const msg = await response.json();
     document.getElementById('randommessage').innerText = msg;
+}
+
+function createMap() {
+
+  var memLatLng = {lat: 35.1495, lng: -90.0490};
+  var gusLatLng = {lat: 35.139368 , lng: -90.057269};
+  var indiaPLatLng = {lat: 35.143319, lng: -90.003102};
+  var centralBLatLng = {lat: 35.126358, lng: -98.985764};
+  var oaoBLatLng = {lat: 35.116002, lng: -89.910065};
+  var ttLatLng = {lat: 35.131409, lng: -89.945915};
+  var etlLatLng = {lat: 35.136951, lng: -89.974301};
+
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: memLatLng, zoom: 11});
+
+  const gusMarker = new google.maps.Marker({
+      position: gusLatLng,
+      map: map,
+      title: "Gus's World Famous Chicken"
```

Comment:
I was exactly at this spot a few years ago! Loved it!
Next time I'm in Memphis, I have some more places to checkout! 😃

Reply:
I swear by all of them! haha